src/V3EmitCSyms.cpp

```diff
@@ -168,9 +168,16 @@ class EmitCSyms final : EmitCBaseVisitor {
     void varHierarchyScopes(string scp) {
         while (!scp.empty()) {
             const auto scpit = m_vpiScopeCandidates.find(scp);
+            // QUESTION -- why do we search for scp but insert scpit->second.m_symName?
+            // in the case of t_vpi_module those are "TOP.t" and "t" respectively
```

Comment:
Obviously I'll remove this comment, but I wanted to remember why this was the way it is.  Is it possible that we should be `find()`ing `scpit->second.m_symName` instead?